pkg/driver/mysql.go

```diff
@@ -0,0 +1,41 @@
+package driver
+
+import (
+	"context"
+	"database/sql/driver"
+	"github.com/go-sql-driver/mysql"
+	"github.com/pkg/errors"
```

Comment:
When doing so, the `errors.Wrap` calls below should be replaced by `fmt.Errorf("wrap wrap %w", ...)`.

Reply:
I'm just complying to the existing codebase. I don't consider it a good idea to mix different ways of doing the same thing.

Reply:
One part of that package we use is that it also records a stack trace when wrapping errors. So it's not really a drop-in replacement.

Reply:
A short `grep` only showed one occurrence of `errors.WithStack` within [`RuntimeUpdates`](https://github.com/Icinga/icingadb/blob/cd96f0de6f33b6750ba44d3934d4380a62ab14a6/pkg/icingadb/runtime_updates.go#L282). Thus, this might easily get replaced. But getting into a discussion about this feels like bikeshedding.

I just wanted to notice that the package in question is now over four years old and was marked dead in late 2021 as _almost_ all functionality is now part of the Go standard library.

And even as lots of other places in the code are still relying on it, imho, we do not have to introduce more usage in new code.

Reply:
`github.com/pkg/errors.Wrap()` also records a stack trace: https://pkg.go.dev/github.com/pkg/errors#Wrap

Reply:
Never mind then, sorry.

Reply:
How does the stdlib provide stack traces like `errors.Wrap()`? It doesn't. `pkg/errors` is fine and replacing it with the sole argument that it is dead makes no sense as it just works and has benefits over the stdlib. Otherwise we would also have to replace `sqlx`.

Reply:
Sorry I'm late to the party, but I had written this comment anyway and wanted to add my two cents.